### ISSUE TITLE
Picker delay

### ISSUE THREAD

**Issue by basequatro** [start-react/native-starter-kit#45]
There's a delay with picker on NSK, tried to downgrade to NB 0.5.7 but i got the same behaviour.
This doesn't occurs with awesome project and nativebase. Already mentioned in issue #44, but was closed without answer.

![pickerlag](https://cloud.githubusercontent.com/assets/2417603/19353981/88d04804-913b-11e6-87a5-9265f6324d62.gif)

The delay increases depending on how many routes we have in application.

``` javascript
import React, { Component } from 'react';
import { connect } from 'react-redux';
import { Container, Header, Title, Content, Text, Button, Icon, Picker } from 'native-base';

import { openDrawer } from '../../actions/drawer';
import { popRoute } from '../../actions/route';
import styles from './styles';

const Item = Picker.Item;

class BlankPage extends Component {

  static propTypes = {
    popRoute: React.PropTypes.func,
    openDrawer: React.PropTypes.func,
    name: React.PropTypes.string,
    index: React.PropTypes.number,
    list: React.PropTypes.arrayOf(React.PropTypes.string),
  }

  constructor(props) {
    super(props);
    this.state = {
      selectedItem: undefined,
      selected1: 'key1',
      results: {
        items: [],
      },
    };
  }

  onValueChange(value: string) {
    this.setState({
      selected1: value,
    });
  }

  popRoute() {
    this.props.popRoute();
  }

  render() {
    const { props: { name, index, list } } = this;

    return (
      <Container style={styles.container}>
        <Header>
          <Button transparent onPress={() => this.popRoute()}>
            <Icon name="ios-arrow-back" />
          </Button>

          <Title>{(name) ? this.props.name : 'Blank Page'}</Title>

          <Button transparent onPress={this.props.openDrawer}>
            <Icon name="ios-menu" />
          </Button>
        </Header>

        <Content padder>
          <Picker
            iosHeader="Select one"
            mode="dropdown"
            selectedValue={this.state.selected1}
            onValueChange={this.onValueChange.bind(this)}
          >
            <Item label="Cats" value="key0" />
            <Item label="Dogs" value="key1" />
            <Item label="Birds" value="key2" />
            <Item label="Elephants" value="key3" />
          </Picker>
        </Content>
      </Container>
    );
  }
}

function bindAction(dispatch) {
  return {
    openDrawer: () => dispatch(openDrawer()),
    popRoute: () => dispatch(popRoute()),
  };
}

function mapStateToProps(state) {
  return {
    name: state.user.name,
    index: state.list.selectedIndex,
    list: state.list.list,
  };
}

export default connect(mapStateToProps, bindAction)(BlankPage);
```

**Comment by basequatro** [comment]
@SupriyaKalghatgi any plans to release a fix for this?

**Comment by SupriyaKalghatgi** [comment]
@basequatro This will be fixed with the next release of NativeBase.
Keep waiting

**Comment by abcd-ca** [comment]
FYI, takes about 30 seconds to show in the Kitchen Sink app from the app store

**Comment by basequatro** [comment]
Solved with Native Base 0.5.13
